packages/cmf/__tests__/sagas/documentTitle.test.js

```diff
@@ -0,0 +1,122 @@
+import SagaTester from 'redux-saga-tester';
+import changeDocumentTitle, {
+	formatPath,
+	buildMapFromRoutes,
+	getTitleFromRoutes,
+	assignDocTitle,
+} from '../../src/sagas/documentTitle';
+
+describe('changeDocumentTitle', () => {
+	it('sagas wich initiliaze after receiving events REACT_CMF.REQUEST_SETTINGS_OK and @@router/LOCATION_CHANGE', () => {
```

Comment:
Make it simpler :)
`it('should change document title on REACT_CMF.REQUEST_SETTINGS_OK and @@router/LOCATION_CHANGE event')`

You could even split it into 2 tests : REACT_CMF.REQUEST_SETTINGS_OK event and @@router/LOCATION_CHANGE event